inc/theme-update/class-astra-theme-background-updater.php

```diff
@@ -397,6 +392,29 @@ public static function update_db_version( $version = null ) {
 			delete_transient( 'astra-addon-db-migrated' );
 
 			do_action( 'astra_theme_update_after' );
+			
+			// This will clear Astra and Astra Pro asset cache on every new update.
+			self::astra_clear_all_assets_cache();
```

Comment:
```suggestion
			astra_clear_all_assets_cache();
```
@sagarb90 Instead of creating a new function, can we call this existing function directly?

Reply:
@premanshup Yes we can use the existing function I thought we have functions in `astra-update-functions.php` file which are called in db update process.